### PR TITLE
Test PHP 7 support

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by dbu** [comment]
* zendframework/zend-feed is unhappy now with php 5.3 and 5.4 . we might just downgrade or simply remove the zend-feed example. or have composer "update" in travis.yml for those versions to downgrade.
* non-existent service "doctrine_phpcr.admin.default_session". Did you mean this: "doctrine_phpcr.default_session"? i think we did mean that, probably came with the upgrade.

**Comment by WouterJ** [comment]
I've fixed the first problem by always using `composer update` in our travis builds, except from the PHP 5.6 job. The second problem was a bug in DoctrinePHPCRBundle and has been fixed.

**Comment by dbu** [comment]
travis complained about not finding symfony-cmf/symfony-cmf 1.3 - i restarted the push build to see if it was a glitch.

i am not sure about the update for non 5.6 versions. if somebody still uses older php versions and follows our doc, he will get the not installable complaint. imo we should drop 5.3 and 5.4 support as they are end of live. somebody starting a new project better use a non-dead php version. we need to be more careful with the bundles to not block upgrades, but for the sandbox i see no big deal in upping the version requirement.

File: composer.json [hunk 1]
```diff
@@ -14,40 +14,42 @@
...
-        "sensio/framework-extra-bundle": "~2.3",
-        "symfony-cmf/symfony-cmf": "1.2.*",
+        "symfony/symfony": "^2.3",
+        "symfony/assetic-bundle": "^2.3",
+        "symfony/swiftmailer-bundle": "^2.3",
+        "symfony/monolog-bundle": "^2.3",
+        "twig/extensions": "1.*",
+        "sensio/distribution-bundle": "3.*",
+        "sensio/framework-extra-bundle": "^2.3",
+        "symfony-cmf/symfony-cmf": "dev-1.3-versions",
```
Comment (lsmith77) [inline composer.json]: I guess this should just be ``1.3@dev`` ?

**Comment by WouterJ** [comment]
Tests are failing because https://github.com/doctrine/DoctrinePHPCRBundle/pull/214 is not yet in a RC release

**Comment by dbu** [comment]
this now can't install at all. the phpcr bundle is tagged.

**Comment by WouterJ** [comment]
https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/pull/350 has to be merged to solve the failures

**Comment by lsmith77** [comment]
EkoFeedBundle dropped PHP 5.3 support in their latest release https://github.com/eko/FeedBundle/commit/90908e6dc0817d7938168347d0493fd14a0219f9

Not sure why its still picking this newer version in the PHP 5.3 build.

**Comment by dbu** [comment]
i think for the sandbox, we should simply drop php 5.3, actually. no need to care about legacy installations for a demo.

**Comment by WouterJ** [comment]
Finally, everything is green.

**Pull request merged**